src/settings/Settings.js

```diff
@@ -188,6 +188,11 @@ export const SETTINGS = {
         displayName: _td('Mirror local video feed'),
         default: false,
     },
+    "TagPanel.disableTagPanel": {
+        supportedLevels: LEVELS_ACCOUNT_SETTINGS,
+        displayName: _td('Disable Community Filter Panel'),
```

Comment:
I know it's a bit late now, but having the settings positively named makes the list feel a lot more manageable (as the user doesn't have to think *too* hard). Setting the default to `true` and having this be `Enable LLP` would achieve the same goal.